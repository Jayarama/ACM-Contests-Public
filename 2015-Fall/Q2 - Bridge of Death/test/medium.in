Bedemir, Robin, Galahad
What is your name?
What is your quest?
What is the capital of Assyria?
What is your name?
What is your favorite color?
What is the capital of Assyria?
What is your name?
What is the capital of Assyria?
What is your favorite color?
Arthur, Galahad
What is your name?
What is the airspeed velocity of an unladen swallow?
What is your quest?
What is your name?
What is the capital of Assyria?
What is your favorite color?
