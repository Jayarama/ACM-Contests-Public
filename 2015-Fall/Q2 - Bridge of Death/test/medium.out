Ser Bedemir of Camelot
I seek the grail
Nineveh
Ser Robin of Camelot
Yellow
Nineveh
Ser Galahad of Camelot
Nineveh
Yellow
Arthur, King of the Britons
African or European?
